shop/src/main/java/ro/msg/learning/shop/dto/OrderDTO.java

```diff
@@ -0,0 +1,21 @@
+package ro.msg.learning.shop.dto;
+
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+import ro.msg.learning.shop.utils.Address;
+
+import java.util.List;
+
+@Data
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+public class OrderDTO {
+    private Integer orderID;
+    private Integer userID;
```

Comment:
orderId. the D is too big. Regardless if you make it capital or not, given that ID is short for identification, it should be consistent throughout your code.